lib/entry-index.js

```diff
@@ -1,20 +1,41 @@
 'use strict'
 
-const asyncMap = require('slide/lib/async-map')
+module.exports = {
+  find,
+  insert,
+  ls,
+  lsStream,
+  delete: del,
+  _hashKey: hashKey,
+  _bucketPath: bucketPath
+}
```

Comment:
I'll back out this change! In the unlikely event we start to run into circular dep issues it's easy enough to apply as a codemod.